.github/workflows/python-test.yml

```diff
@@ -1,34 +1,35 @@
 name: Python Test
 on:
   push:
-    branches: main
+    branches:
+      - main
   pull_request:
-    branches: main
+    branches:
+      - main
 
 jobs:
   build:
     strategy:
       matrix:
         python-version:
           - "3.12"
+          - "3.13"
```

Comment:
👍🏽

Reply:
A second thought on this: in a _someone hit the lotto and is on a long vacation_ situation, it could be a blocker to a patch release.  My ideal would be that py3.13 testing is in a separate job (`python-test-next`) that is not required, but is hugely proactive.

Reply:
I think this gets into whether `cirrus-geo` is expected to be used as a library or is it just an lambda function "app" and a CLI. If not a library, should we look at changing pyproject.toml to target a single Python version rather than `>=`?